### ISSUE TITLE
[Button] button is not working on Outlook

### ISSUE THREAD

**Issue by jeov10** [foundation/foundation-emails#878]
*How can we reproduce this bug?**

`  <row>
    <columns>
      <center>
        <button href="${config.mailing.loginAdmin}" target="_blank" class="large credi-u-fill-color--gray">Ir a ALGO</button>
      </center>
    </columns>
  </row>`

**What did you expect to happen?**
See the blue button.
I can see the button on gmail web and mobile and outlook mobile but **it's not working on outlook web.** 

**What happened instead?**
it's not showing a button just a rectangle with the href info exposed like this "${config.mailing.loginAdmin}Ir a ALGO"

**What email clients does this happen in?**
Outlook web.

**Comment by JoeKolba** [comment]
This might be the solution... Take out the variable in the href and put in a real url to test.  I had the same issue and it was because of the variable in the href.

**Comment by logangarcia** [comment]
Yes, OWA strips placeholder links -- you can use a named anchor (e.g. `#admin-login`) for testing.

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
